examples/coordShiftTest/shift.cc

```diff
@@ -0,0 +1,111 @@
+#include "common/map_coordinate.h"
```

Comment:
Really small point - I think our directory naming convention is lower_snake_case so this should be ``coord_shift_test``

Reply:
:+1: